src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXExtensions.cs

```diff
@@ -31,6 +31,12 @@ public static class SPDXExtensions
     /// </summary>
     public static string GenerateSpdxId(Element element, string id) => $"SPDXRef-{element.Type}-{GetStringHash(id)}";
```

Comment:
General comment--only some of the methods in this class are extension methods (i.e., they use the `this` keyword on the first parameter). It might be good to make at least the new method (not necessarily the old ones) actually be extension methods

Reply:
I will address this in a separate PR